lib/validator.js

```diff
@@ -110,7 +110,21 @@ function validateRepo(r, data, licenses) {
       reportError('incompletew3cjson', {error: "group"});
     } else {
       // Note that `data.groups` is unused here.
-      groups = arrayify(conf.group).map(id => parseInt(id, 10));
+      // conf.groups can be either a list of numeric ids
+      // or a list of strings à la `${grouptype}/${groupshortname}`
+      // cf https://github.com/w3c/validate-repos/issues/107
+      groups = [];
+      for (const groupid of arrayify(conf.group)) {
+        let id = groupid;
+        if (typeof id === "string") {
+          id = parseInt(groupid, 10);
+          // this wasn't a stringified number
+          if (isNaN(id)) {
+            id = w3cgroups.find(g => g.fullshortname === groupid).id;
```

Comment:
this should probably check that the group is found and if not, report an `inconsistentgroups` error.

Reply:
3c51e1a addresses with, but with an invalidw3cjson instead